allauth/socialaccount/providers/quickbooks/views.py

```diff
@@ -32,7 +32,9 @@ def complete_login(self, request, app, token, **kwargs):
     def get_user_info(self, token):
         auth_header = 'Bearer ' + token.token
         headers = {'Accept': 'application/json', 'Authorization': auth_header, 'accept': 'application/json'}
-        if app_settings.PROVIDERS.quickbooks.SANDBOX:
+        QBO_sandbox = getattr(app_settings, 'SOCIALACCOUNT_PROVIDERS', {}).get(
```

Comment:
You can replace this with self.get_provider().get_settings().get('SANDBOX', False)

Reply:
Just figured this out from looking at other providers, thank you!